### PR TITLE
fix(docker): fix wrong format of load_plugins when docker start

### PR DESCRIPTION
**Pull request by zhanghongtong**
Hot fix https://github.com/emqx/emqx-rel/issues/535#issuecomment-653668643

### CODE REVIEW

File: deploy/docker/docker-entrypoint.sh [hunk 1]
```diff
@@ -177,10 +178,11 @@ if [[ ! -z "$EMQX_LOADED_MODULES" ]]; then
     echo "EMQX_LOADED_MODULES=$EMQX_LOADED_MODULES"
     # Parse module names and place `{module_name, true}.` tuples in `loaded_modules`.
     for var in $(echo "$EMQX_LOADED_MODULES"|sed -e "s/^[^A-Za-z0-9_]\{1,\}//g"|sed -e "s/[^A-Za-z0-9_]\{1,\}/\ /g"); do
-        if [ ! -z $(grep -o $var ${_EMQX_HOME}/data/loaded_modules) ]; then
+        if [ ! -z "$(grep -oE "\{($var),[ ]*(true|false)\}" ${_EMQX_HOME}/data/loaded_plugins)" ]; then
```
Comment (louisburton) [inline deploy/docker/docker-entrypoint.sh:181]: isn't this supposed to be `loaded_modules` not `loaded_plugins`?

File: deploy/docker/docker-entrypoint.sh [hunk 2]
```diff
@@ -177,10 +178,11 @@ if [[ ! -z "$EMQX_LOADED_MODULES" ]]; then
...
             echo "$(sed -r "s/\{($var),[ ]*(true|false)\}./\{\1, true\}./1" ${_EMQX_HOME}/data/loaded_modules)" > ${_EMQX_HOME}/data/loaded_modules
+        elif [ ! -z $(grep -o "$var\." ${_EMQX_HOME}/data/loaded_modules) ]; then
             # backward compatible.
-            echo "$(sed -r -e "s/($var)./\{\1, true\}./1" ${_EMQX_HOME}/data/loaded_modules)" > ${_EMQX_HOME}/data/loaded_modules
+            echo "$(sed -r "s/($var)./\{\1, true\}./1" ${_EMQX_HOME}/data/loaded_modules)" > ${_EMQX_HOME}/data/loaded_modules
         else
             echo "{$var, true}." >> ${_EMQX_HOME}/data/loaded_modules
```
Comment (louisburton) [inline deploy/docker/docker-entrypoint.sh:187]: Is this coping with the lack of newline at the end of the template file?
This value `EMQX_LOADED_MODULES="emqx_mod_acl_internal,emqx_mod_presence,emqx_mod_rewrite"`
Leaves me with :
```
opt/emqx $ cat data/loaded_modules
{emqx_mod_acl_internal, true}.
{emqx_mod_presence, true}.{emqx_mod_acl_internal, true}.
{emqx_mod_presence, true}.
{emqx_mod_rewrite, true}.
{emqx_mod_acl_internal, true}.
{emqx_mod_presence, true}.
{emqx_mod_rewrite, true}.
```

File: deploy/docker/docker-entrypoint.sh (hunk 1 above)
Comment (zhanghongtong) [reply deploy/docker/docker-entrypoint.sh:181]: Hi, @louisburton
Thanks for your feedback, I will fix it in https://github.com/emqx/emqx-rel/pull/561 and https://github.com/emqx/emqx-rel/pull/559

File: deploy/docker/docker-entrypoint.sh (hunk 2 above)
Comment (zhanghongtong) [reply deploy/docker/docker-entrypoint.sh:187]: This is caused by the previous problem

**Pull request merged**